rbi/core/kernel.rbi

```diff
@@ -451,33 +451,33 @@ module Kernel
     params(
         initial: T.any(Integer, Float, Rational, BigDecimal, String),
         digits: Integer,
+        exception: T::Boolean
     )
     .returns(BigDecimal)
   end
-  def BigDecimal(initial, digits=0); end
+  def BigDecimal(initial, digits=0, exception: true); end
+  # As of 2.6, all these numeric conversion methods *can* return `nil` iff given the kwarg `exception: false`, which
```

Comment:
could you add a link to this PR as it has valuable discussion?

Reply:
Updated!